go/registry/api/runtime.go

```diff
@@ -179,10 +187,13 @@ type StorageParameters struct {
 
 	// CheckpointChunkSize is the chunk size parameter for checkpoint creation.
 	CheckpointChunkSize uint64 `json:"checkpoint_chunk_size"`
+
+	// MinPoolSize is the minimum required candidate storage node pool size.
+	MinPoolSize uint64 `json:"min_pool_size"`
 }
 
 // ValidateBasic performs basic storage parameter validity checks.
-func (s *StorageParameters) ValidateBasic() error {
+func (s *StorageParameters) ValidateBasic(version uint16) error {
```

Comment:
The version argument should no longer be needed as we only allow version 2 descriptors.